Rakefile

```diff
@@ -14,14 +14,23 @@ DATAROOTDIR = DATADIR = ENV['DATAROOTDIR'] || "#{PREFIX}/share"
 MANDIR = ENV['MANDIR'] || "#{DATAROOTDIR}/man"
 PKGDIR = ENV['PKGDIR'] || File.expand_path('pkg')
 
+exporter = ENV['PATH'].split(':').push(
+  '/usr/bin',
+  ENV['KAFO_EXPORTER']).find do |exporter|
```

Comment:
I don't like the reuse of exporter. Isn't dir (or something similar) a better naming?